TESTS/mbed_hal/common_tickers/main.cpp

```diff
@@ -531,6 +534,19 @@ utest::v1::status_t us_ticker_teardown(const Case *const source, const size_t pa
 #endif
     osKernelResume(0);
 
+    /* Redeem from missing go-off ticker interrupt
+     *
+     * For testing us ticker HAL, we uninstall us ticker layer and then reinstall it back. This would cause ticker
+     * interrupt which goes off but doesn't get caught during the uninstall period. At the end of test, we manually
+     * fire one ticker interrupt for the us ticker layer to redeem from missing go-off ticker interrupt.
+     *
+     * With MBED_TICKLESS and LPTICKER_DELAY_TICKS enabled, lp_ticker_set_interrupt_wrapper, relying on us ticker
+     * layer, would arm an alarm to call lp_ticker_set_interrupt delayed. Due to missing go-off ticker interrupt,
+     * the armed alarm won't go off until another ticker interrupt goes off. Manually firing one ticker interrupt
+     * can alleviate the issue.
+     */
+    us_ticker_fire_interrupt();
```

Comment:
This call to us_ticker_fire_interrupt should no longer be needed with the fix #7524. The call to `ticker_resume` earlier in this function should restart the common ticker layer.

Reply:
@c1728p9 I remove the unnecessary code with #7524 in https://github.com/ARMmbed/mbed-os/pull/8030/commits/4888ff2b1108219f7f027af72b32a649dc5f712a.